src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -14,24 +14,26 @@ public enum CallType
         Method = 1,
         Set = 8,
     }
-    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IList
+    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
     {
         public Collection() { }
         public int Count { get { throw null; } }
+        public object this[int Index] { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(2))]
```

Comment:
>(2) [](start = 100, length = 3)

Nit: why was the System.AttributeTargets below fixed to use the actual enum value and this wasn't?